### ISSUE TITLE
Wrong Proxy Object registering in managers

### ISSUE THREAD

**Issue by joj0s** [dano/aioprocessing#41]
I am unable to use the `AioEvent` class of a process manager. The issue is that in [`managers.py`](https://github.com/dano/aioprocessing/blob/master/aioprocessing/managers.py) there are proxy object classes created for both `AioEvent` and `AioBarrier` but they are never registered with the SyncManager:

```python
AioSyncManager.register("AioBarrier", Barrier, AioQueueProxy)
AioSyncManager.register("AioEvent", Event, AioQueueProxy)
```

I believe these two lines should have registered the `AioBarrierProxy` and `AioEventProxy`respectively.

**Comment by joj0s** [comment]
@dano Thank you very much for merging my fix! Are you planning on publishing a new release soon? We would really like to use your library in the project I am working on and this issue is currently preventing us from doing so.

**Comment by dano** [comment]
@joj0s I just published v2.0.1, which includes this fix.

**Comment by joj0s** [comment]
Appreciate it, thank you!
